src/main/java/io/github/jpmorganchase/fusion/Fusion.java

```diff
@@ -603,6 +610,11 @@ public FusionBuilder rootURL(String rootURL) {
             this.rootURL = rootURL;
             return this;
         }
+
+        public FusionBuilder datasetBearerToken(String catalog, String dataset, String token) {
```

Comment:
Is this the right name for it? I'm not sure of a better one though - Fusion Download Token?

Reply:
Will have a think when I am doing the build for upload/download.